src/commands.cc

```diff
@@ -1173,6 +1174,12 @@ const CommandDesc debug_cmd = {
         {
             profile_hash_maps();
         }
+        else if (parser[0] == "faces")
+        {
+            write_to_debug_buffer("Faces:");
+            for (auto& face : FaceRegistry::instance().aliases())
+                write_to_debug_buffer(format(" * {}: {}", face.key, face_to_str(face.value.face)));
```

Comment:
no need for `face_to_str` here, format should know how to format a face automatically provided a `to_string(Face)` function exists.